app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java

```diff
@@ -294,6 +297,13 @@ public void onPause() {
      * @param locationChangeType defines if location shanged significantly or slightly
      */
     private void refreshView(LocationServiceManager.LocationChangeType locationChangeType) {
+
+        if(!haveNetworkConnection())
+        {
+            Toast.makeText(getApplicationContext(),"No internet connnection ",Toast.LENGTH_LONG).show();
+            hideProgressBar();
+            return;
+        }
         if (lockNearbyView) {
```

Comment:
We have a showLongToast method in our ViewUtils class, please use that one.

Reply:
I couldn't find the showLongToast method there's only a showSnackbar method.

Reply:
Very weird I have merged a PR recently which adds it.

Reply:
I'll check again

Reply:
Oh, actually PR removes it, sorry for confusion. You can etiher use snackbar to display network problem, or create a method for Toast in ViewUtils. SnackBar can be better according to em, but your decision

Reply:
ok thanks for the clarification :-)
I'll use the snackbar.